Explorer/Assets/DCL/EmojiPanel/EmojiPanelView.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using DCL.UI;
+using UnityEngine;
+using UnityEngine.Serialization;
+using UnityEngine.UI;
+
+namespace DCL.Emoji
+{
+    public class EmojiPanelView : MonoBehaviour
+    {
+        public event Action<EmojiSectionName, bool> OnSectionSelected;
+        [SerializeField] public List<EmojiSectionToggle> emojiSections;
+
+        [SerializeField] public ScrollRect scrollView;
+        [SerializeField] public Transform emojiContainer;
+        [SerializeField] public Transform emojiContainerScrollView;
+        [SerializeField] public Transform emojiSearchResults;
+        [SerializeField] public Transform emojiSearchedContent;
+        [SerializeField] public SearchBarView searchPanelView;
+
+        public event Action OnEmojiFirstOpen;
```

Comment:
Can we achieve the same without a callback? Don't we know when the view gets open?

Reply:
Will change in a separate PR for the cleaning of the design where most likely I'll just remove this